xml/System.Numerics/IFloatingPoint`1.xml

```diff
@@ -351,7 +351,7 @@
         <param name="x">The value to round.</param>
         <param name="digits">The number of fractional digits to which <paramref name="x" /> should be rounded.</param>
         <param name="mode">The mode under which <paramref name="x" /> should be rounded.</param>
-        <summary>Rounds a value to a specified number of fractional-digits using the default rounding mode (<see cref="F:System.MidpointRounding.ToEven" />).</summary>
+        <summary>Rounds a value to a specified number of fractional-digits using the specified rounding mode.</summary>
```

Comment:
```suggestion
        <summary>Rounds a value to a specified number of fractional digits using the specified rounding mode.</summary>
```

Reply:
Umm, you might want to update all of the other ones if you've done this. Including on .NET Runtime repo.